src/Conference/ConferenceDetails.jsx

```diff
@@ -1,8 +1,9 @@
 import React, { useEffect, useState } from "react";
 import styled from "styled-components";
 import { getTimestring, getDatestring } from "../utils";
-import { Link, useParams } from "react-router-dom";
-import { getConferenceById } from "../api.js";
+import { Link, useParams, useHistory } from "react-router-dom";
```

Comment:
Bit of a nit: Usually you'd order your imports: 

1. External libraries
2. Your components
So I'd put react-router-dom before the utils import :)